src/Client.js

```diff
@@ -323,6 +338,27 @@ class Client extends EventEmitter {
    *   console.log(products[0].productId); // INK_SACK:3
    * })
    * .catch(console.log);
+   */ /**
+   * Allows you to get bingo data
+   * @method
+   * @name Client#getSkyblockBingo
```

Comment:
```suggestion
   * @name Client.skyblock#getBingo
```
(that's clearly not how you do it is it)